deploy/helm/kubecf/assets/operations/releases.yaml

```diff
@@ -1,95 +1,19 @@
-{{- $releases = append $releases "capi" }}
-{{- $releases = append $releases "cf-networking" }}
-{{- $releases = append $releases "cf-smoke-tests" }}
-{{- $releases = append $releases "cflinuxfs3" }}
-{{- if .Values.features.credhub.enabled }}
-  {{- $releases = append $releases "credhub" }}
-{{- end }}
-{{- $releases = append $releases "diego" }}
-{{- $releases = append $releases "dotnet-core-buildpack" }}
-{{- $releases = append $releases "garden-runc" }}
-{{- $releases = append $releases "go-buildpack" }}
-{{- $releases = append $releases "java-buildpack" }}
-{{- $releases = append $releases "loggregator" }}
-{{- $releases = append $releases "nats" }}
-{{- $releases = append $releases "nginx-buildpack" }}
-{{- $releases = append $releases "r-buildpack" }}
-{{- $releases = append $releases "nodejs-buildpack" }}
-{{- $releases = append $releases "php-buildpack" }}
-{{- $releases = append $releases "python-buildpack" }}
-{{- $releases = append $releases "routing" }}
-{{- $releases = append $releases "ruby-buildpack" }}
-{{- $releases = append $releases "silk" }}
-{{- $releases = append $releases "staticfile-buildpack" }}
-{{- $releases = append $releases "statsd-injector" }}
-{{- $releases = append $releases "uaa" }}
-{{- $releases = append $releases "loggregator-agent" }}
-{{- $releases = append $releases "log-cache" }}
-{{- $releases = append $releases "cf-cli" }}
-
-# Add the cf-acceptance-tests-release.
-{{- if .Values.testing.cf_acceptance_tests.enabled }}
-{{- $releases = append $releases "cf-acceptance-tests" }}
-- type: replace
-  path: /releases/-
-  value:
-    name: cf-acceptance-tests
-{{- end }}
-
-{{- if .Values.testing.brain_tests.enabled }}
-{{- $releases = append $releases "brain-tests" }}
-- path: /releases/-
-  type: replace
-  value:
-    name: brain-tests
-{{- end }}
+  path: /releases
 
-# SITS only makes sense when using Diego, for this reason, we only enable it if Eirini is not
-# enabled.
-{{- if and .Values.testing.sync_integration_tests.enabled (not .Values.features.eirini.enabled) }}
-{{- $releases = append $releases "sync-integration-tests" }}
-- path: /releases/-
-  type: replace
-  value:
-    name: sync-integration-tests
-{{- end }}
-
-# Add app-autoscaler related releases.
-{{- if .Values.features.autoscaler.enabled }}
-{{- $releases = append $releases "app-autoscaler" }}
-- type: replace
-  path: /releases/-
-  value:
-    name: app-autoscaler
-{{- $releases = append $releases "postgres" }}
-- type: replace
-  path: /releases/-
-  value:
-    name: postgres
-{{- end }}
-
-# Loop over the releases, replacing the url and stemcell, and the version when appropriate.
-{{- range $release := $releases }}
+{{- $_ := include "_config.lookup" (list $ "releases") }}
+{{- range $release_name, $release := $.kubecf.retval }}
+  {{- /* only add bosh releases (releases with a version) */}}
+  {{- if and $release.version (eq "true" (include "_config.condition" (list $ (index $release "condition")))) }}
```

Comment:
This may be easier to understand as nested `if`s.

Reply:
Done in 59d6969